src/commands/api/getDatabase.ts

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { DocDBDatabaseTreeItem } from '../../../src/docdb/tree/DocDBDatabaseTreeItem';
```

Comment:
nit: `'../../../src/docdb/tree/DocDBDatabaseTreeItem'` is equivalent to `'../../docdb/tree/DocDBDatabaseTreeItem'`

In other words you can remove '../src' from the middle of the path. Same comment in the getConnectionString file